discord/ext/commands/converter.py

```diff
@@ -441,3 +441,24 @@ def repl2(obj):
 
         pattern = re.compile('|'.join(transformations.keys()))
         return pattern.sub(repl2, result)
+
+class Union(Converter):
+    """Tries to convert the argument to each of a list of
+    converters, in order.
+
+    Attributes
+    ------------
+    converters : tuple(:class:`Converter`,any)
```

Comment:
This should probably point to the documentation on converters.

Unfortunately it doesn't have a ref yet so it's not possible atm.